.github/workflows/container-version.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  schedule:
+    - cron: '37 13 * * *'
+name: busybox-update workflow
+jobs:
+  checkVersionAndCreatePR:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run busybox updater
+        run: |
+          chmod +x ./scripts/busybox-updater.sh
+          ./scripts/busybox-updater.sh
+        shell: bash
+      - name: Create Pull Request
```

Comment:
How about adding a step after this to log the PR link using this: https://github.com/peter-evans/create-pull-request#action-outputs? It can then be referencable from [Action run](https://github.com/thanos-io/thanos/actions). 🙂

Reply:
Yea I'm not sure what you mean. 
What I do think is that we should move this forward, and you make an iteration with your magic on it. You have some good ideas and know how to make this better. No need to be limited by my work :) WDYT?

Reply:
Sure, we can totally iterate! Would be nice to also get community input as well once this is in action! 🤗